### PR TITLE
Update and fix bugs for python sdk

### PR DESCRIPTION
**Pull request by kakakikikeke-fork**
I created a pull request for the dev branch again.  
Please check the following for details.

* https://github.com/Netflix/conductor/pull/530
* https://github.com/Netflix/conductor/pull/529
* https://github.com/Netflix/conductor/pull/514
* https://github.com/Netflix/conductor/pull/513

### CODE REVIEW

**Comment by seenureddy** [comment]
@hunterford  Can you assign reviewers?

File: client/python/conductor/conductor.py [hunk 1]
```diff
@@ -267,6 +283,41 @@ def restartWorkflow(self, wfId, taskRefName, fromTaskRef):
...
+        BaseClient.__init__(self, baseURL, self.BASE_RESOURCE)
+
+    def getEventHandlerDef(self, event, activeOnly=True):
+        url = self.makeUrl('{}', event)
+        params = {}
+        params['activeOnly'] = activeOnly
+        return self.get(url, params)
+
+    def getEventHandlerDefs(self):
+        url = self.makeUrl('')
```
Comment (zsimic) [inline client/python/conductor/conductor.py]: Minor: you could give 1st argument a default value so you don't have to pass an empty format, like so:
```
    def makeUrl(self, urlformat=None, *argv):
        url = self.baseResource + '/'
        if urlformat:
             url += urlformat.format(*argv)
        return url
```

**Comment by kakakikikeke-fork** [comment]
Thank you reviewed. I will check and fix them.

**Comment by seenureddy** [comment]
@zsimic Can you merge this branch?

**Pull request merged**